### ISSUE TITLE
Add some more documentation

### ISSUE THREAD

**Issue by youngchief-btw** [jellz/minecraft-server#20]

**Comment by antoine4livre** [comment]
Yes ! I don't understand how it works 😢

**Comment by youngchief-btw** [comment]
Yeah, I noticed you have to see the code and try to figure out what means what

**Comment by youngchief-btw** [comment]
I don't know if he will even notice the issue......

**Comment by jellz** [comment]
I noticed :smile: I kinda stopped working on this project. If you have questions about the code that you'd like me to answer feel free to message me on Discord (daniel#0004).
I might get back into the project, not really sure yet, sorry to disappoint.
Cheers!

**Comment by youngchief-btw** [comment]
Oh Okay, Well that cleared up many things about the inactivity

**Comment by IlanLuci** [comment]
Hello! I really love this project and would love to help in any way I can. Do you think you will end up getting back to this? I added you on discord. My username is MooMan.

**Comment by jellz** [comment]
Hey, my Discord account is actually daniel2#1000 now. I can't access my
other account (daniel#0004).

**Comment by IlanLuci** [comment]
Ok. Thanks. I added you there.

**Comment by youngchief-btw** [comment]
k
